ui/components/app/modals/confirm-remove-account/confirm-remove-account.stories.js

```diff
@@ -6,12 +6,6 @@ export default {
   id: __filename,
   component: ConfirmRemoveAccount,
   argTypes: {
-    hideModal: {
```

Comment:
Looks like `hideModal` is an accepted prop, but `chainId` and `rpcPrefs` are not. Those are set by the container.

Reply:
Though... `hideModal` _ought_ to be set by the container. Seems like a bug that it's missing.

Reply:
Updated in [49e3187](https://github.com/MetaMask/metamask-extension/pull/14331/commits/49e3187d5a0274d4442ca59684512cf386f49035)

Reply:
I did not fix the hideModal bug though